### ISSUE TITLE
docs: browsers support

### ISSUE THREAD

**Issue by samerset-rmn** [toss/es-toolkit#800]
Hi! 

I really like this library, but I can't find out which browsers it supports and which it doesn't. My guess is that support for older browsers is not a primary goal here, and I understand that. But it would be great to have this information somewhere in the docs – a list of supported browsers or a note warning about it.

**Comment by raon0211** [comment]
Yes, it should be supported in browsers that support ES6, as long as there's proper polyfill support. However, let's check the browser compatibility to be sure.

**Comment by Schweinepriester** [comment]
I arrive here via https://mastodon.social/@ecmascript_news@mastodon.online/114930466356931569 and would be very intrigued to replace our lodash usage, but was also looking for 

> a list of supported browsers or a note warning about it

but didn't find it 😅
